js_modules/dagit/packages/core/src/assets/usePartitionNameForPipeline.tsx

```diff
@@ -6,6 +6,7 @@ import {RepoAddress} from '../workspace/types';
 
 export function usePartitionNameForPipeline(repoAddress: RepoAddress, pipelineName: string) {
   const {data: partitionSetsData} = useQuery(ASSET_JOB_PARTITION_SETS_QUERY, {
+    fetchPolicy: 'cache-first', // todo bengotow: Is this correct?
```

Comment:
I actually think cache-first is pretty reasonable for this one, but this also isn't an expensive query so I'd vote to let it do cache-and-network